pkg/machines/vmdiskstab.jsx

```diff
@@ -0,0 +1,116 @@
+/*jshint esversion: 6 */
```

Comment:
good catch, this change should be part of https://github.com/cockpit-project/cockpit/pull/6072/commits/1d775591b41dd5b23cdf20376ffd9f3e50905eb8